static-site/src/components/splash/showcase.yaml

```diff
@@ -1,31 +1,37 @@
 cards:
-- name: SARS-CoV-2 (GISAID)
+- name: SARS-CoV-2
   img: ncov.png
-  url: /ncov/gisaid/global
-- name: SARS-CoV-2 (open)
-  img: ncov.png
-  url: /ncov/open/global
-- name: Avian influenza (cattle flu genome)
+  url: /ncov/
+- name: H5N1 cattle outbreak
   img: avianinfluenza.png
   url: /avian-flu/h5n1-cattle-outbreak/genome
-- name: Avian influenza (H5N1/HA)
+- name: avian influenza
   img: avianinfluenza.png
   url: /avian-flu/h5n1/ha/2y
-- name: Mpox (clade IIb)
+- name: mpox
   img: mpox.png
   url: /mpox/clade-IIb
-- name: DRC Ebola (2018-19)
+- name: Ebola in the DRC
   img: ebola2.png
   url: /community/inrb-drc/ebola-nord-kivu
-- name: Measles (N450)
+- name: measles
   img: measles.jpg
   url: /measles/N450
-- name: Seasonal Influenza
+- name: seasonal influenza
   img: seasonalinfluenza.png
   url: /seasonal-flu
+- name: RSV
+  img: rsv1.png
+  url: /rsv
+- name: Yersinia pestis
+  img: yersinia.png
+  url: /community/ktmeaton/yersinia-pestis/maximum-likelihood/all?m=div
 - name: WNV in the Americas (narrative)
   img: wnv2.png
   url: /narratives/twenty-years-of-WNV
-- name: DRC Ebola Sit Rep (narrative)
-  img: ebola3.png
-  url: /narratives/inrb-ebola-example-sit-rep
+- name: Chikungunya
+  img: chikv.png
+  url: /groups/ViennaRNA/CHIKVnext
```

Comment:
_typography nit_

@trvrb wrote:

> I also switched to capitalization consistent with mid-sentence usage, eg "mpox" rather than "Mpox" as per our capitalization on the /pathogens page.

When I saw this capitalization on preview, I thought, "That ain't right", and I went to go suggest a patch:

```diff
diff --git a/static-site/src/components/splash/showcase.yaml b/static-site/src/components/splash/showcase.yaml
index f08765f1..9b643166 100644
--- a/static-site/src/components/splash/showcase.yaml
+++ b/static-site/src/components/splash/showcase.yaml
@@ -5,19 +5,19 @@ cards:
 - name: H5N1 cattle outbreak
   img: avianinfluenza.png
   url: /avian-flu/h5n1-cattle-outbreak/genome
-- name: avian influenza
+- name: Avian influenza
   img: avianinfluenza.png
   url: /avian-flu/h5n1/ha/2y
-- name: mpox
+- name: Mpox
   img: mpox.png
   url: /mpox/clade-IIb
 - name: Ebola in the DRC
   img: ebola2.png
   url: /community/inrb-drc/ebola-nord-kivu
-- name: measles
+- name: Measles
   img: measles.jpg
   url: /measles/N450
-- name: seasonal influenza
+- name: Seasonal influenza
   img: seasonalinfluenza.png
   url: /seasonal-flu
 - name: RSV
```

…and then I saw that it was intentional. :-/

The current casing here seems inconsistent to me and looked sloppy when I saw it. Why title case "Chikungunya" but lower case "seasonal influenza"? Or why title case "Ebola in the DRC" instead of "ebola in the DRC"?

(I also don't get why we'd want to aim for "mid-sentence usage" instead of title/heading usage since these are titles/headings after all.)

Reply:
Merging the PR now but I agree and will make a follow-up issue. Things seem inconsistent both on the splash page and `/pathogens`.

Reply:
Continuing in #917